daprdocs/content/en/dotnet-sdk-docs/dotnet-development/dotnet-development-docker-compose.md

```diff
@@ -0,0 +1,46 @@
+---
+type: docs
+title: "Dapr .NET SDK Development with Docker-Compose"
+linkTitle: "Development"
+weight: 50000
+description: Learn about local development with Docker-Compose
+---
+
+## Docker-Compose
+
+*Consider this to be a .NET companion to the [Dapr Self-Hosted with Docker Guide]({{ ref self-hosted-with-docker.md }}))*.
+
+`docker-compose` is a CLI tool included with Docker Desktop that you can use to run multiple containers at a time.
+
+`docker-compose` is a way to automate the lifecycle of multiple containers together, and offers a development experience similar to a production environment for applications targeting Kubernetes.
+
+**Pro:** Since `docker-compose` manages containers for you, you can make dependencies part of the application definition and stop the long-running containers on your machine.
+
+**Con:** most investment required, services need to be containerized to get started.
+
+**Con:** can be difficult to debug and troubleshoot if you are unfamilar with Docker.
```

Comment:
```suggestion
- **Pro:** Since `docker-compose` manages containers for you, you can make dependencies part of the application definition and stop the long-running containers on your machine.
- **Con:** most investment required, services need to be containerized to get started.
- **Con:** can be difficult to debug and troubleshoot if you are unfamilar with Docker.
```